application/deposit/deposit_test.go

```diff
@@ -130,7 +130,7 @@ func TestDeposit(t *testing.T) {
 			t.Fatal(err)
 		}
 		//utxoIDs, retVal, err := hndlr.GetValueForOwner(txn, testingOwner(), 2)
```

Comment:
remove dead code comment

Reply:
Just to be sure, going forward, when we change a function, would you like us to remove dead code comments that were already present pre-fork? Because there are a lot more